### DISCUSSION TITLE
Wompi Colombian payment gateway

### DISCUSSION THREAD

**Discussion by giucloud** [juspay/hyperswitch#5230] [Ideas / Feature Requests]
Allow you to add wompi payment methods in Colombia, to natively integrate Colombian payment methods

**Comment by giucloud** [comment]
https://docs.wompi.co/docs/colombia/metodos-de-pago/

**Comment by gorakhnathy7** [comment]
Hey @giucloud, Thanks for bringing this to our notice!
While the region in which Wompi operates is of distinguished importance to us, and we will look forward to get them onboarded. But, being a project driven with open source principles, Can you help us with more inputs, to help us make a well informed decision around this - 
What is the approximate market coverage by Wompi in the Columbian region? 
Is there a specific payment related use-case in which Wompi excel in this region? 
Are you a merchant already integrated with Wompi, and would like to contribute in their onboarding with Hyperswitch?

Looking forward to hear from you!

**Reply by giucloud** [reply]
The Colombian market is around 45 million people with Internet access, in Colombia we have 3 main payment methods, cash (efecty) bank transfer (pse) and cards, unlike other Colombian payment processors, wompi allows through its API to integrate these payment methods without the need to go to our checkout, allowing native integrations in addition to low fees, In addition to integrating native payment with nequi (most used wallet) and sumaspay (credit payment) all natively, the project already integrates some methods such as efecty or pse through zen, but zen is inaccessible to Colombian businesses and its rate is too high

**Reply by giucloud** [reply]
In addition to other local payment methods natively using APIs, wompi offers very good rates and technological tools and APIs for good integration and use, they are the best  rates

**Reply by giucloud** [reply]
IsAs far as I know, it is the only gateway that allows you to integrate your payment methods via API, without the need to go to an external payment screen

**Reply by giucloud** [reply]
And taking advantage of them they can integrate wompi Panama and wompi el salvador to cover payments in those areas, their documentation is identical so it would be easy to integrate it and allow them to cover those markets

**Comment by gorakhnathy7** [comment]
Thanks for this @giucloud, will keep you posted with any updates we have around it.

**Reply by giucloud** [reply]
I would like to contribute

**Reply by giucloud** [reply]
@gorakhnathy7 Hello friend, do you know if my proposal has been considered?
